pages.es/osx/dockutil.md

```diff
@@ -0,0 +1,32 @@
+# dockutil
+
+> Gestiona los elementos del dock de macOS.
+> Más información: <https://github.com/kcrawford/dockutil>.
+
+- Añade una aplicación al final del dock del usuario actual:
+
+`dockutil --add {{ruta/a/la/aplicación}}`
+
+- Sustituye una aplicación por otra en el dock del usuario actual:
+
+`dockutil --add {{ruta/a/la/aplicación}}} --replacing '{{etiqueta_de_elemento_del_dock}}'`
+
+- Añade un directorio con opciones de visualización y lo muestra como un icono de carpeta o pila:
```

Comment:
```suggestion
- Añade un directorio con opciones de visualización y muestralo como un icono de carpeta o pila:
```

Reply:
`muestralo` is not properly conjugated verb. The proper way is `lo muestra`, `lo visualiza`, `lo presenta`.